watrank/src/components/autofill-popup.tsx

```diff
@@ -34,65 +34,62 @@ const CommandKey: React.FC<CommandKeyProps> = ({ text }) => {
     );
 };
 
+interface AutofillPopupContentProps {
+    children?: React.ReactNode;
+    refresh?: () => void;
+}
+
+function AutofillPopupContent({
+    children,
+    refresh,
+}: AutofillPopupContentProps) {
+    return (
+        <DialogContent>
+            <DialogHeader>
+                <DialogTitle>Import jobs</DialogTitle>
+            </DialogHeader>
+            <ul className="list-decimal pl-6 leading-loose">
+                <li>
+                    Go to your applications table on{" "}
+                    <a
+                        href="https://waterlooworks.uwaterloo.ca/myAccount/co-op/full/applications.htm"
+                        target="_blank"
+                        className="underline"
+                    >
+                        WaterlooWorks
+                    </a>
+                </li>
+                <li>
+                    Select all using <CommandKey text="A" /> then copy with{" "}
+                    <CommandKey text="C" />
+                </li>
+                <li>Click the button below!</li>
```

Comment:
Fixed this feedback

<img width="751" alt="Screenshot 2025-02-14 at 4 54 39 PM" src="https://github.com/user-attachments/assets/4c0fb3f8-ee50-4fec-ab60-a28d42b9e539" />